google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/ITDatastoreTest.java

```diff
@@ -853,9 +856,10 @@ public Integer run(DatastoreReaderWriter transaction) {
   }
 
   @Test
-  public void testSkippedResults() {
+  public void testSkippedResults() throws InterruptedException {
     Query<Key> query = Query.newKeyQueryBuilder().setOffset(Integer.MAX_VALUE).build();
     int numberOfEntities = DATASTORE.run(query).getSkippedResults();
+    Thread.sleep(1000);
     assertEquals(2, numberOfEntities);
   }
```

Comment:
this sleep is redundant. it can't affect the value of numberOfEntities

Reply:
done

Reply:
Looks good!